app/src/main/java/org/mozilla/focus/viewmodel/ShoppingSearchPromptViewModel.kt

```diff
@@ -3,17 +3,27 @@ package org.mozilla.focus.viewmodel
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import org.mozilla.focus.telemetry.TelemetryWrapper
 import org.mozilla.rocket.download.SingleLiveEvent
 import org.mozilla.rocket.shopping.search.domain.GetShoppingSitesUseCase
 import org.mozilla.rocket.shopping.search.ui.adapter.ShoppingSiteItem
 
 class ShoppingSearchPromptViewModel(getShoppingSite: GetShoppingSitesUseCase) : ViewModel() {
+    private val matchedShoppingSiteTitle = MutableLiveData<String?>()
+
     val promptVisibilityState = MutableLiveData<VisibilityState>()
-    val matchedShoppingSiteTitle = MutableLiveData<String?>()
     val shoppingSiteList: LiveData<List<ShoppingSiteItem>> = getShoppingSite()
 
     val openShoppingSearch = SingleLiveEvent<Unit>()
 
+    fun openShoppingSearch() {
```

Comment:
will that be better to name it as `onShoppingSearchPromptButtonClicked` or something like that? Since it will also send the pin `clickTabSwipeDrawer`.

Reply:
Done